docs/hub/spaces.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Documentation for Spaces
+---
+
+<h1>How to get started with Spaces</h1>
+
+> ⚠️ This feature is currently in private beta, reach out to [julien-c](https://huggingface.co/julien-c) if you'd like to try it out ⚠️
```

Comment:
I don't think https://huggingface.co/julien-c provides any email to contact you, should this be your email instead?